workbench/kibana.json

```diff
@@ -1,6 +1,6 @@
 {
   "id": "opendistroQueryWorkbench",
```

Comment:
in odfe 1.12 workbench plugin changed to `opendistroQueryWorkbenchKibana`, could you update here?

Reply:
Yes, will do